File: engine/model/src/main/scala/tools/packet/owl/GetOwlOpen.scala
package tools.packet.owl

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GetOwlOpen(private var _leaderboards: java.util.List[java.lang.Integer]) extends PacketInput {
  def leaderboards: java.util.List[java.lang.Integer] = _leaderboards

  override def opcode(): SendOpcode = SendOpcode.SHOP_SCANNER_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/owl/GetOwlOpen.java
package tools.packet.owl;

import java.util.List;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GetOwlOpen(List<Integer> leaderboards) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOP_SCANNER_RESULT;
   }
}